src/helpers/helpers.segment.js

```diff
@@ -190,7 +191,7 @@ function solidSegments(points, start, max, loop) {
 			if (!prev.skip) {
 				loop = false;
 				result.push({start: start % count, end: (end - 1) % count, loop});
-				start = last = cur.stop ? end : null;
+				last = cur.stop ? end : null;
```

Comment:
`last` and `start` can be different at line 206, this should break something.

Reply:
Hmm. This one I was super difficult for me to try to figure out how to get it to pass the check. Do you think you'd be able to take a stab at this one since you're more familiar with this section of code?

Reply:
// @ts-ingore would be my solution here.
Other options would be to add another local nullable variable to keep the `start` in or allow nullable `start` as input. Bot are kind of wrong IMO.

Reply:
I can't get `@ts-ignore` to work here for some reason

Reply:
I figured it out. I copy / pasted from above where it was "ingore" instead of "ignore". It's working now